src/python/gudhi/off_utils.pyx

```diff
@@ -110,11 +110,12 @@ def read_points_from_off_file(off_file=''):
     """
     with open(off_file) as input_file:
         dim, nb_points = _read_off_file_header(input_file)
-        print(dim, nb_points)
         # usecols=list(range(dim)) stands here to avoid comments at the end of line
         # or colors that can be added in RGB format after the points, the faces, ...
-        return np.loadtxt(input_file, dtype=np.floating, comments='#',
-                          usecols=list(range(dim)), max_rows=nb_points)
+        points = np.loadtxt(input_file, dtype=np.floating, comments='#',
```

Comment:
It isn't very clear what dtype np.floating yields (it is documented as "Abstract base class of all floating-point scalar types."). np.float64 or np.double would be clearer.

Reply:
Clarifcation on 8d012cc5e